CDCgov/prime-reportstream
# update specimen types for covid-19 and sender-automation valuesets

This PR adds multiple specimen types to covid-19 and sender-automation valuesets:

Test Steps:
1. Pull down MT production settings and load to local database with ./prime multiple-settings get -f mt -a -l --env prod
    make sure to run ./prime login --env prod first to get an admin token
2.) use ./prime data --input-schema upload-covid-19 --input <test-upload-covid-19.csv file-location>  --output-schema covid-19 --format HL7 --output-dir <directory-to-output>
 use ./prime data --input-schema PrimeDataInput/pdi-covid-19  --input <test-pdi-covid-19.csv file-location>  --output-schema covid-19 --format HL7 --output-dir <directory-to-output>

ensure that SPM-4 is appropriately filled out in all files.

[test-upload-covid-19.csv](https://github.com/CDCgov/prime-reportstream/files/15003848/test-upload-covid-19.csv)

[test-pdi-covid-19.csv](https://github.com/CDCgov/prime-reportstream/files/15003845/test-pdi-covid-19.csv)

## Changes
covid-19/specimen-type:

- code: 1293160008
 display: Swab from middle region of inferior nasal meatus

sender-automation/specimen-type

- code: 1293160008
   display: Swab from middle region of inferior nasal meatus
- code: 1293160008
   display: 1293160008
- code: 119342007
   display: Saliva specimen
- code: 119342007
   display: 119342007


## Checklist

### Testing

## Linked Issues
- Fixes #14074
